src/js/frontend/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Do we really want to disable this?

Reply:
Yes, there are some, but very few cases where we want the `any` type especially for plugins when we won't necessarily know the type before they're uploaded. I'm going to be very conscious about avoiding this in any situation possible, but we might need it in certain situations. We can switch it back for this PR, but we'll eventually need an any type,